src/test/skript/tests/syntaxes/effects/EffBlockUpdate.sk

```diff
@@ -0,0 +1,8 @@
+test "block update":
+	set {_loc} to spawn of world "world" ~ vector(10,10,10)
+	set {_blocks::*} to blocks in radius 2 of block at {_loc}
+
+	update blocks at {_blocks::*} as sand without physics updates
+	assert blocks at {_blocks::*} is sand with "1 or more block did not update with or without physics"
+	update blocks at {_blocks::*} as air without physics updates
+	assert blocks at {_blocks::*} is air with "1 or more block did not update to air"
```

Comment:
```suggestion
	assert blocks at {_blocks::*} is air with "1 or more block did not update to air"

```

Reply:
Not sure why, but it's not displaying the blank/newline after it on github. But shows for me on IntelliJ